plugins/outputs/stackdriver/stackdriver.go

```diff
@@ -69,6 +56,18 @@ var sampleConfig = `
   #   location = "eu-north0"
 `
 
+// Stackdriver is the Google Stackdriver config info.
+type Stackdriver struct {
+	Project        string
+	Namespace      string
```

Comment:
Can you please add TOML tags for those two options.

Reply:
👍